src/Mailer/Email.php

```diff
@@ -2060,7 +2060,8 @@ public function send($content = null)
 
         return $contents;
     }
-
+    
```

Comment:
Whitespace issue, needs to be reverted

Reply:
You mean I could only add a line instead of removing and add two lines?

Reply:
You also added whitespace characters here. Best to make sure modified lines are trimmed "right".